test/integration/deploy/channels/slack/test.verify-slack.js

```diff
@@ -112,4 +150,13 @@ describe('deploy verify-slack integration tests', () => {
       .update('authorize')
       .digest('hex');
   }
+
+  function validatePipelineCreation(pipelineName) {
+    const supplierWsk = openwhisk({
```

Comment:
you might want to consider extracting this initialization out so the openwhisk object doesn't get initialized every time this method gets called.